ethereum-verifier/Readme.md

```diff
@@ -0,0 +1,9 @@
+# Ethereum Stark Verifier
+## Install
```

Comment:
empty line between headings. 

Here's the markdown style guide: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#rules

Here's a VSCode plugin that implements it: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint